src/Modules/Station/Components/AddSong/PreviewVideo/PreviewVideo.tsx

```diff
@@ -39,18 +41,24 @@ export class PreviewVideo extends Component<any, any> {
   }
 
   public handleButtonAddSong() {
+    if (this.state.isAdding) {
+      return;
+    }
+
     this.props.addSong(this.state.message);
+    this.setState({ isAdding: true });
   }
 
   public clearMessage() {
```

Comment:
If you do that, you should change this method name to: Ex: ResetPreviewState or sth for readable!